scripts/publish-docker-images

```diff
@@ -127,12 +129,31 @@ push_multi-arch_manifest_docker_hub() {
 		rm multi-arch-temp.yaml
 	done
 
+	# Create digest files
+	if [[ ! -z "${FILE_PATH}" ]]; then
+		if [[ -d "${FILE_PATH}" ]]; then
```

Comment:
> we'd be able to have them in a single file and extract as needed. 

We have them in single file. The output of `manifest-tool inspect --raw "${IMAGE_NAME}:${IMAGE_TAG_LATEST}"` gives us all the required digest. Output JSON is in the form -> [{},{},{}] . 

The first entry is always corresponds to the multi arch image. For the second and third entry in JSON we have to check the _architecture_ flag to check which entry is which(ARM or AMD).
By having three separate files we can avoid doing that check and always pick up the digest of the first entry in the JSON 

> let's rename this variable along with its flag usage

Agree with this. Will do.

Reply:
The code currently writes to *multiple files* in the path that user provides where each has the image's inspected manifest. The merged json with key structure I proposed is instead results in a *single file* that the user specified - which, again, is what I had in mind when we had discussed ways to pass this data along.  

So that the user doesn't have a whole directory to enumerate and transfer if needed, they could instead have a single file with the structured data in the json, this doesn't change that the first element in each value is the "magic" manifest element. 

The "magic" `manifest.list` entry is otherwise lacking any other identifier than its "MediaType"; the keying I suggest doesn't change this magical state of affairs (you'd use something like `.["amazon/amazon-ecs-agent:latest"][0].Digest` to get to that "magic" element). This also eliminates the metadata maintained in the filename for this use case.